packages/core/types/index.js

```diff
@@ -285,6 +285,11 @@ export type DetailedReportOptions = {|
   assetsPerBundle?: number,
 |};
 
+export type WatcherOptions = {|
+  ...BaseWatcherOptions,
+  ignore: BaseWatcherOptions['ignore'],
```

Comment:
huh, neat, I didn't realise you could reference part of a type like that with Flow!

Reply:
I was hoping this would make it non-nullable, but apparently, the preferred way is to wrap the type with `$NonMaybeType<MaybeType>`. It's not as neat though, the vscode server doesn't display the particular type.